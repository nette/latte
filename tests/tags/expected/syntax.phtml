<?php
%A%
final class Template%a% extends Latte\Runtime\Template
{

	public function main(): array
	{
%A%
		echo '<ul>
';
		$iterations = 0;
		foreach ($people as $person) /* line %d% */ {
			echo '	<li>';
			echo LR\Filters::escapeHtmlText(($this->filters->upper)($person)) /* line %d% */;
			echo '</li>
';
			$iterations++;
		}
		echo '{* comment latte *}
</ul>

<p>Default: ';
		echo LR\Filters::escapeHtmlText($person) /* line %d% */;
		echo '</p>

<p>Default: ';
		echo LR\Filters::escapeHtmlText($person) /* line %d% */;
		echo '</p>

<p>Default: {$person}</p>


{a}
{ {/ {/syntax
';
		return get_defined_vars();
	}


	public function prepare(): void
	{
%A%
	}

}
