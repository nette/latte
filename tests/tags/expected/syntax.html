<ul>
	<li>JOHN</li>
	<li>MARY</li>
	<li>PAUL</li>
{* comment latte *}
</ul>

<p>Default: Paul</p>

<p>Default: Paul</p>

<p>Default: {$person}</p>

{a}
{ {/ {/syntax
